From 744e74f493957046246dae27824ea1a5334abf36 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Sat, 3 Jan 2026 14:12:17 +0800
Subject: [PATCH 54/63] loonggpu-bridge: remove redundant bridge func vptr
 assignment

On Linux >= 6.16, devm_drm_bridge_alloc() already assigns it.

Signed-off-by: Xi Ruoyao <xry111@xry111.site>
---
 loonggpu-bridge/bridge_phy.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/loonggpu-bridge/bridge_phy.c b/loonggpu-bridge/bridge_phy.c
index 195441d..e0e5eb5 100644
--- a/loonggpu-bridge/bridge_phy.c
+++ b/loonggpu-bridge/bridge_phy.c
@@ -373,7 +373,9 @@ static int bridge_phy_bind(struct loonggpu_bridge_phy *phy)
 {
 	int ret;
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 16, 0)
 	phy->bridge.funcs = &bridge_funcs;
+#endif
 	drm_bridge_add(&phy->bridge);
 	ret = lg_drm_bridge_attach(phy->encoder, &phy->bridge, NULL, 0);
 	if (ret) {
-- 
2.52.0

